config/secrets_example.yml

```diff
@@ -14,11 +14,23 @@ development:
   secret_key_base: secret
   admin_user: dev
   admin_password: devpwd
+  newrelic:
```

Comment:
Yes, it is repetitive to repeat these in dev and test, but this repo (and asis) run CI against ruby 3.1.3 and 3.2.1 and those checks failed when I attempted to use aliases here because of https://github.com/rails/rails/issues/44209 Failing checks: https://app.circleci.com/pipelines/github/lorawoodford/i14y/149/workflows/cea17e78-8599-4862-8982-381b5f9ecaf6

We're likely in for a fair bit of work to fix this across our repos when we move to 3.1+ so I've deferred for now in order to keep this upgrade moving forward.